tests/scripts/run-throughput-perf.py

```diff
@@ -290,6 +295,11 @@ def runIterations(dll_name, dll_path, iterations, crossgen_path, jit_path, assem
     if opt_level == 'min_opt':
         my_env['COMPlus_JITMinOpts'] = '1'
 
+    if jit_name == 'legacy':
+        my_env['COMPLUS_AltJit'] = '*'
```

Comment:
According to my results on PerfLab legacy was the fastest one, then ryujit, and legacy+minopt is the slowest.